Libs/Visualization/VTK/Widgets/ctkVTKAbstractView.h

```diff
@@ -82,6 +82,16 @@ public Q_SLOTS:
   /// \sa scheduleRender
   virtual void forceRender();
 
+  /// Calls forceRender if the rendering has not bee paused from pauseRender()
+  /// \sa pendingRender
+  virtual void pendingRender();
+
+  /// Prevents scheduleRender() from calling forceRender
+  /// Applications are responsible for managing pauseRender called so that each
+  /// pauseRender(true) is matched with a pauseRender(false)
```

Comment:
We should rather write that caller is responsible to restore previous state of pauseRender (instead of assuming previous state was false). Also add an example, just in case:

    \code
    bool wasPaused = view->pauseRender(true);
    // Perform operations that may call view->scheduleRender().
    view->pauseRender(wasPaused);
    \endcode